extensions/datasource/runtime/src/main/java/io/quarkus/datasource/runtime/DevServicesBuildTimeConfig.java

```diff
@@ -32,4 +33,12 @@
      */
     @ConfigItem
     public Map<String, String> properties;
+
+    /**
+     * Optional fixed port the dev service will listen to.
+     * <p>
+     * If not defined, the port will be chosen randomly.
+     */
+    @ConfigItem
+    public OptionalInt port;
```

Comment:
Are we planning to add the same capability for Mongo dev services https://github.com/quarkusio/quarkus/blob/2cd3b3b4a7d489614bf1e3e8e46f359af5d94723/extensions/mongodb-client/deployment/src/main/java/io/quarkus/mongodb/deployment/DevServicesBuildTimeConfig.java#L9?

Reply:
I added a commit with it.

Reply:
Awesome, thanks!